src/views/Alteration.vue

```diff
@@ -48,7 +46,17 @@
         @haveChanges="onAlterationSummaryChanges()"
       />
 
-      <no-fee-summary class="mt-10" />
+      <documents-delivery
+       class="mt-10"
+       :pleaseValidate="true"
+        @valid="setDocumentOptionalEmailValidity($event)"
```

Comment:
Good. Now _eventually_ there will need to be a method that checks all the individual validity flags to determine whether the alteration as a whole is valid. As that method doesn't exist yet (I think?) then you have nothing to update there; otherwise you would.

I believe this is a future validation ticket. No change needed here. FYI only